packages/rexray/extra/dcos-rexray.service

```diff
@@ -8,4 +8,8 @@ RestartSec=15
 LimitNOFILE=16384
 EnvironmentFile=/opt/mesosphere/environment
 Environment=REXRAY_HOME=/
+# REX-Ray fails to clean up stale lock files by itself.
+ExecStartPre=/bin/rm -f /var/run/libstorage/lsx.lock
+# Allow REX-Ray configuration to be changed during upgrade.
```

Comment:
Indeed - I installed a 1.9.0 cluster, then upgraded only one of the agents to this PR, created a task on an old agent (mesos containerizer) that writes some text to an external volume, killed that task, created a new task constrained to the upgraded agent (docker containerizer), confirmed that the text could be read from the same external volume. I then added more text to the external volume from the upgraded agent, and confirmed that the changes were visible from an app started on the unupgraded agent.

I did this test starting with an unupgraded agent, as well as starting from the upgraded agent.